From 5f25de59673621c4270cc2158a7a5a6de9aa8b5f Mon Sep 17 00:00:00 2001
From: Will Thompson <will@willthompson.co.uk>
Date: Mon, 15 Jul 2019 21:11:01 +0100
Subject: [PATCH 3/6] appdata: simplify <content_rating>

The default level for each category is 'none'.

Spotted by Patrick Griffis over at
<https://github.com/flathub/org.gnome.Gnote/pull/8#discussion_r303571194>.
---
 data/appdata/gnote.appdata.xml.in | 30 +-----------------------------
 1 file changed, 1 insertion(+), 29 deletions(-)

diff --git a/data/appdata/gnote.appdata.xml.in b/data/appdata/gnote.appdata.xml.in
index e3d49da6..851a5926 100644
--- a/data/appdata/gnote.appdata.xml.in
+++ b/data/appdata/gnote.appdata.xml.in
@@ -20,34 +20,6 @@
  <url type="donation">http://www.gnome.org/friends/</url>
  <url type="translate">https://wiki.gnome.org/TranslationProject</url>
  <update_contact>aurisc4_at_gmail.com</update_contact>
- <content_rating type="oars-1.1">
-    <content_attribute id="violence-cartoon">none</content_attribute>
-    <content_attribute id="violence-fantasy">none</content_attribute>
-    <content_attribute id="violence-realistic">none</content_attribute>
-    <content_attribute id="violence-bloodshed">none</content_attribute>
-    <content_attribute id="violence-sexual">none</content_attribute>
-    <content_attribute id="violence-desecration">none</content_attribute>
-    <content_attribute id="violence-slavery">none</content_attribute>
-    <content_attribute id="violence-worship">none</content_attribute>
-    <content_attribute id="drugs-alcohol">none</content_attribute>
-    <content_attribute id="drugs-narcotics">none</content_attribute>
-    <content_attribute id="drugs-tobacco">none</content_attribute>
-    <content_attribute id="sex-nudity">none</content_attribute>
-    <content_attribute id="sex-themes">none</content_attribute>
-    <content_attribute id="sex-homosexuality">none</content_attribute>
-    <content_attribute id="sex-prostitution">none</content_attribute>
-    <content_attribute id="sex-adultery">none</content_attribute>
-    <content_attribute id="sex-appearance">none</content_attribute>
-    <content_attribute id="language-profanity">none</content_attribute>
-    <content_attribute id="language-humor">none</content_attribute>
-    <content_attribute id="language-discrimination">none</content_attribute>
-    <content_attribute id="social-chat">none</content_attribute>
-    <content_attribute id="social-info">none</content_attribute>
-    <content_attribute id="social-audio">none</content_attribute>
-    <content_attribute id="social-location">none</content_attribute>
-    <content_attribute id="social-contacts">none</content_attribute>
-    <content_attribute id="money-purchasing">none</content_attribute>
-    <content_attribute id="money-gambling">none</content_attribute>
- </content_rating>
+ <content_rating type="oars-1.1"/>
 </application>
 
-- 
2.20.1

